test/elasticsearch.yml

```diff
@@ -16,27 +16,21 @@ spec:
     spec:
       containers:
         - name: elasticsearch
-          image: docker.elastic.co/elasticsearch/elasticsearch:5.6.0
+          image: docker.elastic.co/elasticsearch/elasticsearch:5.6.12
```

Comment:
I have bumped patch version. And disabled xpack, we should not use it anyways. There an official ES6x without licensed xpack, but I would rather keep using the lowest supported version.